datasets/fr/tresor/fr_tresor_gels_avoir.yml

```diff
@@ -311,6 +311,58 @@ lookups:
       - match: "Société mère: Rostec"
         associates:
           - Rostec
+      - match: 'Principaux établissements: Émirats arabes unis, Fédération de Russie, Union européenne'
+        props:
+          jurisdiction: 
+            - ae
+            - ru
+            - eu
+      - match: "Lieu d'enregistrement: Moscou, Fédération de Russie (125080, Moscou, sh. Volokolamskoe, 1 str. 1, et 5; pom VI; kom 30B"
+        props:
+          jurisdiction: ru
+          address: "Moscou, Fédération de Russie (125080, Moscou, sh. Volokolamskoe, 1 str. 1, et 5; pom VI; kom 30B"
+      - match: 'Lieu des activités: Ukraine, Fédération de Russie'
+        props:
+          jurisdiction: 
+            - ua
+            - ru
+      - match: 'Principal établissement: Fédération de Russie, Pays-Bas'
+        props:
+          jurisdiction: 
+            - ru
+            - nl
+      - match: "Lieu d'enregistrement: Russie"
+        props:
+          jurisdiction: ru
+          address: "Russie"
+      - match: "Type d'entité: société à responsabilité limitée (SARL) (LLC)"
+        props:
+          legalForm: société à responsabilité limitée (SARL) (LLC)
+      - match: 'Principal établissement: Arsenyev, Fédération de Russie'
+        props:
+          jurisdiction: ru
+          address: "Arsenyev, Fédération de Russie"
+      - match: "Lieu d'enregistrement: Dubaï, Émirats arabes unis (Unit OT 17-32, Level 17, Central Park Offices, Dubai International Financial Centre, Dubaï, 507065, Émirats arabes unis)"
+        props:
+          jurisdiction: ae
+          address: "Dubaï, Émirats arabes unis (Unit OT 17-32, Level 17, Central Park Offices, Dubai International Financial Centre, Dubaï, 507065, Émirats arabes unis)"
+      - match: "Lieu d'enregistrement: Fédération de Russie (606083, région de Nijni Novgorod, district de Volodarsky, Mulino)"
+        props:
+          jurisdiction: ru
+          address: "Fédération de Russie (606083, région de Nijni Novgorod, district de Volodarsky, Mulino)"
+      # Not sure how to map the following identifiers:
```

Comment:
So, I think this is https://www.opensanctions.org/entities/NK-WxftW3Bgif7M2bF9NvrKLo/. 

I'm also not sure. It looks like the same info as on the Monaco list, but mangled. We could also additionally just put `note: true` on all of these to not lose the identifiers?